streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -452,7 +475,7 @@ public void shouldThrowLockExceptionIfIOExceptionCaughtWhenTryingToLockStateDir(
             public boolean lockGlobalState(final int retry) throws IOException {
                 throw new IOException("KABOOM!");
             }
-        });
+        }, null);
```

Comment:
Should this be null? It seems that `GlobalStateMangerImpl` never expects it to be null.